src/main/webapp/script.js

```diff
@@ -34,8 +33,8 @@ function listElections() {
 
         // Match the OCD Division ID with the regular expressions to check if this election
         // is a state election, district election, or national election.
-        let stateMatch = stateRegex.exec(ocdId);
-        let districtMatch = districtRegex.exec(ocdId);
+        let stateMatch = regex.exec(ocdId);
+        let districtMatch = regex.exec(ocdId);
```

Comment:
I think you can just combine these into one variable right? since they have the same exact result :)